backend/rp2email.py

```diff
@@ -17,9 +22,23 @@
 logger = logging.getLogger(__name__)
 
 TEMP_DIR = "blob_downloads"
-EMAIL_ENDPOINT = '/api/reports/email'
 PDF_OUTPUT_NAME = "report.pdf"
-AZURE_FUNCTION_URL = f"{os.getenv('AZURE_FUNCTION_URL')}{os.getenv('AZURE_FUNCTION_CODE')}"
+HTML_TO_PDF_ENDPOINT = os.getenv('ORCHESTRATOR_URI') + "/api/html_to_pdf_converter"
+
+def get_secret(secretName):
```

Comment:
There's another definition for this function in app.py, but I did not want to create a file to avoid replicating code today, before the demo.